app/src/main/index.js

```diff
@@ -237,12 +238,20 @@ async function startLCD(home, nodeIP) {
       nodeIP,
       "--chain-id",
       chainId
-      // '--trust-node'
     ])
     logProcess(child, join(home, "lcd.log"))
 
-    // XXX: should wait for server to be ready
-    setTimeout(() => resolve(child), 1000)
+    // poll until LCD is started
+    let client = new LcdClient(`http://localhost:${LCD_PORT}`)
```

Comment:
nice and simple solution.
would in a perfect world it be better if the LCD properly tells us when it is ready to serve?